repl/src/main/scala/ammonite/repl/Util.scala

```diff
@@ -124,7 +124,8 @@ case class Catching(handler: PartialFunction[Throwable, Res.Failing]) {
 }
 
 case class Evaluated(wrapper: Seq[Name],
-                     imports: Imports)
+                     imports: Imports,
+                     tag: String = "")
```

Comment:
Shouldn't use a default argument here; explicitly provide it in the places where it's necessary